ntbs-ui-tests/Hooks/TestConfig.cs

```diff
@@ -19,7 +19,7 @@ public TestConfig()
         // This allows JS adding an element into a page to do so before Selenium claims it is not there; it reduces
         // the probability of a false-failure due to Selenium and JS on the site racing each other.
         // If you are getting unexpected "element not found" errors, try increasing this timespan.
-        public TimeSpan ImplicitWait => TimeSpan.FromSeconds(5);
+        public TimeSpan ImplicitWait => TimeSpan.FromSeconds(7);
```

Comment:
I don't think there would be much harm in raising this to 10 or even 15 seconds - it ought not make the tests much slower. The load tests suggest that page loads are very rarely slower than this.

Reply:
The other thing I thought is that maybe we should change the cron job so that the tests run out of hours (currently they're at 9am). Maybe we should run them in the evening, before any of the nightly jobs (e.g. 9pm?)

Reply:
This timeout isn't a page load time, it's for elements to appear (for example if JS is adding elements to a page). I can easily make the page load time the same, 10/15 seconds, though?